### ISSUE TITLE
package.html files should be converted to package.java

### ISSUE THREAD

**Issue by freemo** [opennars/opennars-gui#23]
We currently have Package.html files mingled in with the source to contain package level javadocs. This should be done with Package.java files instead.

**Comment by freemo** [comment]
@TonyLo1 still 6 instances left in `automenta.vivisect` package.. this may not matter if we rip the package out, or may be handled when we integrate the package in if we go that route. But lets make sure the ticket stays open until some resolution comes about.

**Comment by TonyLo1** [comment]
Yes - I left automenta.vivisect because of the reasons you stated - I'll leave the ticket open
